### ISSUE TITLE
dotnet CLI fails to fetch v3.1.3 packages with nuget on Kubuntu 20.04

### ISSUE THREAD

**Issue by 610th** [dotnet/sdk#11576]
This might be a nuget issue and not a dotnet CLI issue.

I'm using dotnet-sdk 3.1 from [snap](https://snapcraft.io/dotnet-sdk).

```
> dotnet --version
3.1.201
> dotnet build
Microsoft (R) Build Engine version 16.5.0+d4cbfca49 for .NET Core
Copyright (C) Microsoft Corporation. All rights reserved.
error NU1102: Unable to find package Microsoft.EntityFrameworkCore.SqlServer with version (>= 3.1.3)
error NU1102: Unable to find package Microsoft.EntityFrameworkCore.Design with version (>= 3.1.3)
error NU1102:   - Found 46 version(s) in nuget [ Nearest version: 3.0.0-preview6.19304.10 ]
```
Other 3.1.3 packages work. I was able to solve it manually by downloading the failing packages from [nuget.org](url) and then adding them to the global local storage with the following command:

```
> nuget init packages/ ~/.nuget/packages/ -Expand
Installing Microsoft.EntityFrameworkCore.Design 3.1.3.
Successfully added package 'packages/microsoft.entityframeworkcore.design.3.1.3.nupkg' to feed '~/.nuget/packages/'.
Installing Microsoft.EntityFrameworkCore.SqlServer 3.1.3.
Successfully added package 'packages/microsoft.entityframeworkcore.sqlserver.3.1.3.nupkg' to feed '~/.nuget/packages/'.
```

**Comment by julielerman** [comment]
Hey , I've been struggling with this on one of my MacBooks. On the other it's not a problem. Same thing except 3.1.8 (though I've tried other versions). I have uninstalled all versions of .net core sdk and reinstalled latest. I've checked the Nuget.Config files in ~/.config and ~/.nuget. I've put a Nuget.Config in the solution. I tried some other solutions and have the same problem. This MacBook had safari and the working one has Catalina so I even updated the OS.
I can't understand why it is specifically this package. But it's definitely something on this computer but I just can't figure it out.
